### ISSUE TITLE
Crash when operating with infinity 

### ISSUE THREAD

**Issue by karloluiten** [scoutant/rpn#1]
Steps:
1) 2 [enter][enter][enter][enter]
2) Press x^n button until stack is empty. Now there is an infinity sign
3) Enter
4) x^n again
5) Crash

Like a new RPN app though, nice work!

**Comment by scoutant** [comment]
True. Obviously app will need a stronger infinity sign management.
